test/resources/postman/PSR.postman_collection.json

```diff
@@ -1,9 +1,10 @@
 {
 	"info": {
-		"_postman_id": "a435773b-8698-4b2b-979b-075aebe2e292",
+		"_postman_id": "b3b1316d-bdd9-40d1-8b70-88d258135ca0",
 		"name": "PSR",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
-		"_exporter_id": "1174017"
+		"_exporter_id": "17267595",
+		"_collection_link": "https://restless-rocket-104352.postman.co/workspace/HMRC~9e5089b4-0cdb-47d6-9ee5-ad032f917396/collection/17267595-b3b1316d-bdd9-40d1-8b70-88d258135ca0?action=share&source=collection_link&creator=17267595"
```

Comment:
In addition to @kberec's comment above, did you miss to add anything or this has been added here unintentionally ?

Reply:
I think it would be better if we have surrender related things in our postman script.